Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -12,6 +12,14 @@
 
 # TODO remove these with parameterized prototypes/whatever we end up doing
 # Windoors (alphabetical)
+- type: entity
+  parent: Windoor
+  id: WindoorServiceLocked
```

Comment:
it says alphabetical and yet you have not placed it in the proper spot.

Reply:
done